R/bundlePackage.R

```diff
@@ -157,3 +157,14 @@ validatePackageSource <- function(pkg) {
 
   return()
 }
+
+appUsesR <- function(quartoInfo) {
```

Comment:
Should `appUsesR` and `appUsesPython` float together?

Reply:
Maaaaaybe? But I think you could equally argue that they should live near where they are used. I suspect these will get refactored out at some point in the future (i.e. to become `usesR` and `usesPython` fields of `appMetadata`) so I'm not too worried.